### PR TITLE
added env name to new page

### PR DESCRIPTION
**Pull request by joshkay10**
**What's in this PR?**
Display the environment names for pipeline events

Screenshot 2024-01-23 at 2 05 11 PM

But what if theres way too many environments?
Screenshot 2024-01-23 at 3 04 51 PM

### CODE REVIEW

File: app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx [hunk 1]
```diff
@@ -174,6 +174,10 @@ export const connectedStateCellIcon = css`
 
 export const connectedStateCellEvent = css`
 	margin-left: ${token('space.100')};
+    max-width: 400px;
```
Comment (joshkay10) [inline app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx:177]: 400 felt the most balanced when resizing and using massive amounts of env names... but if there was a standard happy to change!

File: app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx [hunk 2]
```diff
@@ -174,6 +174,10 @@ export const connectedStateCellIcon = css`
...
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```
Comment (rachellerathbone) [inline app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx:180]: You have an example of what this looks like when the contents hits the edge?

Comment (joshkay10) [reply app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx:180]: yup i will put up a pic

File: app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectedJenkinsServers.tsx [hunk 3]
```diff
@@ -92,6 +92,7 @@ const ConnectedJenkinsServers = ({ connectedJenkinsServer }: ConnectedStateProps
 								</>
 								<div className={cx(connectedStateCell, connectedStateCellEvent)}>
 									{mapLastEventStatus(pipeline.lastEventStatus)} {pipeline.lastEventType}
+									{' '}{pipeline.environmentNames?.join(', ')}
```
Comment (yvonnevictoria) [inline app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectedJenkinsServers.tsx:95]: /nit and potentially a design question but do we want to wrap it in brackets? so it becomes
`Successful deployment (stg, prod, etc)`

**Comment by yvonnevictoria** [comment]
Love seeing all the updates to our app :)

**Review by yvonnevictoria — APPROVED** [review]

**Pull request merged**